src/types.ts

```diff
@@ -1,6 +1,7 @@
 import * as vscode from "vscode"
 import { basename } from "path"
 import { existsSync } from "fs"
+import { threadId } from "worker_threads"
```

Comment:
Did this get added by accident?

Reply:
I did not type it. Some tool must have added it. Removing...